src/main/java/simplifier/controllers/UserController.java

```diff
@@ -0,0 +1,26 @@
+package simplifier.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import simplifier.model.User;
+import simplifier.services.UserService;
+
+@RestController
+@RequestMapping("/user")
+public class UserController {
+
+  private UserService userService;
+
+  @Autowired
+  public void setUserService(UserService userService) {
+    this.userService = userService;
+  }
+
+  @GetMapping("/new")
```

Comment:
No need to use additional "/new" path. You  can have the same url but different HTTP methos, e.g. GET for getting users, POST for creating new users and PUT for updating